apps/vr-tests-react-components/.storybook/main.js

```diff
@@ -1,9 +1,38 @@
+// @ts-check
+
 const path = require('path');
-const { TsconfigPathsPlugin } = require('tsconfig-paths-webpack-plugin');
 
-module.exports = /** @type {import('../../../.storybook/main').StorybookBaseConfig} */ ({
-  addons: ['@fluentui/react-storybook-addon'],
+const { registerTsPaths, registerRules, rules, loadWorkspaceAddon } = require('@fluentui/scripts-storybook');
+const tsConfigPath = path.resolve(__dirname, '../../../tsconfig.base.json');
 
+module.exports = /** @type {import('../../../.storybook/main').StorybookBaseConfig} */ ({
+  addons: [
+    {
+      name: 'storybook-addon-swc',
+      options: /** @type {import('storybook-addon-swc').StoryBookAddonSwcOptions} */ ({
```

Comment:
there's no `config => {/** return new config */}` API?

Reply:
there's not. 

ideal would be if it could consume it from static swcrc which we gonna have in place for libraries soon